clients/drcachesim/simulator/cache.cpp

```diff
@@ -37,13 +37,15 @@
 bool
```

Comment:
Normally there'd be a test added with any new feature -- I assume that's coming with the config file change?  It's worth explicitly mentioning that in the commit message.

Reply:
Yes, tests will be added in a subsequent change.